Packs/PrismaCloud/Playbooks/playbook-Prisma_Cloud_-_Network__API_and_Anomaly_Incidents.yml

```diff
@@ -0,0 +1,1999 @@
+              simple: alert.verdict
+            iscontext: true
+          right:
+            value:
+              simple: malicious
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 2570
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "17":
+    id: "17"
+    taskid: 900a4aff-dd98-4611-8b66-b6a56945a6fd
+    type: title
+    task:
+      id: 900a4aff-dd98-4611-8b66-b6a56945a6fd
+      version: -1
+      name: Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "21"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 840,
+          "y": 2780
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "20":
+    id: "20"
+    taskid: fd02b167-e582-4040-8a9f-302fe467f7be
+    type: playbook
+    task:
+      id: fd02b167-e582-4040-8a9f-302fe467f7be
+      version: -1
+      name: Cloud Response - Generic
+      description: |-
+        This playbook provides response playbooks for:
+        - AWS
+        - Azure
+        - GCP
+
+        The response actions available are:
+        - Terminate/Shut down/Power off an instance
+        - Delete/Disable a user
+        - Delete/Revoke/Disable credentials
+        - Block indicators
+      playbookName: Cloud Response - Generic
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "46"
+    scriptarguments:
+      cloudProvider:
+        complex:
+          root: ${
+          accessor: '}'
+          transformers:
+          - operator: If-Elif
+            args:
+              conditions:
+                value:
+                  simple: |-
+                    [
+                      {
+                        "condition": "#{alert.cloudprovider} == 'AWS'",
+                        "return": "AWS"
+                      },
+                      {
+                        "condition": "#{alert.cloudprovider} == 'AZURE'",
+                        "return": "AZURE"
+                      },
+                      {
+                        "condition": "#{alert.cloudprovider} == 'GCP'",
+                        "return": "GCP"
+                      }
+                    ]
+              flags: {}
+      sourceIP:
+        complex:
+          root: DBotScore.Indicator
+          filters:
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: DBotScore.Score
+                iscontext: true
+              right:
+                value:
+                  simple: "3"
```

Comment:
Please verify the source IP input in the Cloud Response - Generic sub-playbook to block the IP on Azure and AWS sub-playbooks.